applications/asset_tracker/CMakeLists.txt

```
@@ -13,6 +13,10 @@ set(PM_STATIC_YML_FILE
   ${CMAKE_CURRENT_SOURCE_DIR}/configuration/${BOARD}/pm_static.yml
   )
 
+list(APPEND mcuboot_OVERLAY_CONFIG
+  "${CMAKE_CURRENT_SOURCE_DIR}/overlay-rsa.conf"
```

I suggest to name the overlay file `mcuboot_overlay-rsa.conf` to make it visible that this overlay file is intended for the mcuboot image, and not that app image.
```suggestion
  "${CMAKE_CURRENT_SOURCE_DIR}/mucboot_overlay-rsa.conf"
```

---

Added